packages/components/src/components/filter-type-group/filter-accordion/filter-accordion.scss

```diff
@@ -0,0 +1,70 @@
+@use "~@infineon/design-system-tokens/dist/tokens";
+
+
+.header {
+  display: flex;
+  align-items: flex-start;
+  align-self: stretch;
+}
+
+.text-and-icon {
+  display: flex;
+  padding: tokens.$ifxSpace150 tokens.$ifxSpace200 tokens.$ifxSpace150 tokens.$ifxSpace250;
+  align-items: center;
+  gap: tokens.$ifxSpace200;
+  flex: 1 0 0;
+  border-bottom: 1px solid tokens.$ifxColorEngineering200;
+
+  &.expanded {
```

Comment:
The addition of this borders causes a movement. I am not sure if this should be prevented or not. If yes, then one solution is to add a transparent border.

Reply:
(y) will look into this today

Reply:
the borders are described like this in the figma file though. Which movement do they cause? Can you send a screenshot?

Reply:
<img width="172" alt="1" src="https://github.com/Infineon/infineon-design-system-stencil/assets/108734092/eeb00917-e1e3-4490-ad1d-2dc849e30900">
<img width="180" alt="2" src="https://github.com/Infineon/infineon-design-system-stencil/assets/108734092/c046c873-445a-4b01-bf3f-469b26ddf239">

Reply:
so whats the movement here?